docs/archive-a-classroom.md

```diff
@@ -1,19 +1,20 @@
 ## Archiving a classroom
 
-You can archive classrooms that you no longer plan on using. If a classroom is archived, you cannot create new assignments or make changes to existing ones. Students will be unable to accept invitations to assignments in archived classes as well. A classroom can be unarchived at any time.
+You can archive classrooms that you no longer plan on using. If a classroom is archived, you cannot create new assignments or make changes to existing ones. Students will be unable to accept invitations to assignments in archived classrooms as well. A classroom can be unarchived at any time.
 
 ### Archiving from the classroom settings
 
 1. [Sign in to GitHub Classroom](https://classroom.github.com/login).
 2. Select the classroom you wish to archive.
-3. Navigate to _Classroom Settings_ within GitHub Classroom.
-4. Scroll down and click **Archive Classroom**.
+3. Navigate to _Classroom settings_ within GitHub Classroom.
+4. Scroll down and click the _Archive classroom_ button.
 ![archive classroom button](/images/help/archiving/archive-button.png)
-5. A modal will open with additional details. Click **Archive Classroom** on the modal to confirm.
+5. A modal will open with additional details. Click _Archive classroom_ on the modal to confirm.
 ![archive classroom modal](/images/help/archiving/archive-modal.png)
 
 ### Archiving from the classroom listing page
 1. [Sign in to GitHub Classroom](https://classroom.github.com/login).
-2. Open the dropdown menu for the classroom you'd like to archive by clicking on the three dots
-3. Click on **Archive Classroom**
-![classroom dropdown menu](/images/help/archiving/archive-drop-down.png)
\ No newline at end of file
+2. From the [classrooms home page](https://classroom.github.com/classrooms), invoke the options menu for the classroom you'd like to archive by clicking on the three dots.
+![archive classroom ellipsis](/images/help/archiving/archive-ellipsis.png)
+3. Select _Archive this classroom_.
```

Comment:
I believe the option in that dropdown menu is `Archive classroom` not `Archive this classroom`. `Archive this classroom` didn't fit the width of the dropdown menu and displayed as `Archive this classro...`. This uses the details-menu Primer component which has a fixed width of 160px. 

If we want to have it say `Archive this classroom`, we can inline a style and override the Primer class settings. I'm guessing that, where possible, we want to use default Primer components to ensure we get the benifits of them being both on brand and built to be responsive. Is this a case where we want to do an override? (Or, perhaps there's a proper Primer way to make the details menu wider and I'm just missing it).